azext_iot/central/commands_device.py

```diff
@@ -303,6 +355,241 @@ def get_command_history(
     )
 
 
+def get_module_command_history(
+    cmd,
+    app_id: str,
+    device_id: str,
+    module_name: str,
+    command_name: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.ga_2022_05_31.value,
+) -> dict:
+    provider = CentralDeviceProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    return provider.get_module_command_history(
+        device_id=device_id,
+        module_name=module_name,
+        component_name=None,
+        command_name=command_name,
+        central_dns_suffix=central_dns_suffix,
+    )
+
+
+def get_module_component_command_history(
+    cmd,
+    app_id: str,
+    device_id: str,
+    module_name: str,
+    component_name: str,
+    command_name: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.ga_2022_05_31.value,
+) -> dict:
+    provider = CentralDeviceProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    return provider.get_module_command_history(
+        device_id=device_id,
+        module_name=module_name,
+        component_name=component_name,
+        command_name=command_name,
+        central_dns_suffix=central_dns_suffix,
+    )
+
+
+def get_module_properties(
+    cmd,
+    app_id: str,
+    device_id,
+    module_name,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.ga_2022_05_31.value,
+) -> dict:
+    provider = CentralDeviceProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    return provider.get_module_properties(
+        device_id=device_id,
+        module_name=module_name,
+        central_dns_suffix=central_dns_suffix,
+    )
+
+
+def replace_module_properties(
```

Comment:
update?

Reply:
This is calling `provider.replace_module_properties` for completely replacing module properties. while `update_module_properties` is only for partially patch.